swagger/oauth.yaml

```diff
@@ -80,6 +85,8 @@ definitions:
     properties:
       access_token:
         type: "string"
+      refresh_token:
+        type: "string"
```

Comment:
@jcdyer I just pushed/squashed an updated YAML file with the list of required fields (everything except refresh token, to your point). all good?